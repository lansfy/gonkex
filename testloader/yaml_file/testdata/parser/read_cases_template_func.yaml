- name: loader MUST call registered testFun template function during case values apply
  method: POST
  path: /some/path
  request: "{{ testFun .foo }} - postfix"
  response:
    200: "result"
  cases:
    - requestArgs:
        foo: 'value 1'
    - requestArgs:
        foo: 'value 2'
