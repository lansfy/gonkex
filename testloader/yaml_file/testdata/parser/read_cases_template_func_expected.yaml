- GetName: "loader MUST call registered testFun template function during case values apply #1"
  GetMethod: POST
  GetRequest: "testfun(value 1) result - postfix"
  Path: /some/path
  GetResponses:
    200: "result"
  GetFileName: testdata/parser/read_cases_template_func.yaml
  GetLineNumber: 1
  OneOfCase: true
  FirstTestInFile: true

- GetName: "loader MUST call registered testFun template function during case values apply #2"
  GetMethod: POST
  Path: /some/path
  GetRequest: "testfun(value 2) result - postfix"
  GetResponses:
    200: "result"
  GetFileName: testdata/parser/read_cases_template_func.yaml
  GetLineNumber: 1
  OneOfCase: true
  LastTestInFile: true
